src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write your code here
+const listPopulation = document.querySelectorAll('.population');
+const listNumber = [...listPopulation].map(item =>
+  +item.textContent.split(',').join(''));
+const total = listNumber.reduce((a, b) => a + b, 0);
+
+document.querySelector('.total-population').replaceWith(
+  total.toLocaleString('en-US'));
```

Comment:
such code isn't clear for reading
will be better to create separated variable for refactored total value and then use this variable here in `replaceWith` method